chirpstack-concentratord-sx1302/src/config/mod.rs

```diff
@@ -124,6 +127,7 @@ impl Default for Gateway {
             region: None,
             model: "".into(),
             model_flags: vec![],
+            gateway_id: "0000000000000000".into(),
```

Comment:
The default should be an empty string: `"".into(),`.

Reply:
Changed to `None `because `gateway_id `is now an `Option`.